sql/core/src/test/resources/sql-tests/results/pgSQL/date.sql.out

```diff
@@ -198,17 +198,29 @@ struct<DATE '1999-01-18':date>
 -- !query 21
 SELECT date '1999 Jan 08'
 -- !query 21 schema
-struct<DATE '1999-01-01':date>
+struct<>
 -- !query 21 output
-1999-01-01
+org.apache.spark.sql.catalyst.parser.ParseException
+
+Cannot parse the DATE value: 1999 Jan 08(line 1, pos 7)
+
+== SQL ==
+SELECT date '1999 Jan 08'
```

Comment:
Note that this is only for `date` prefix. With this PR, we will return `NULL` like Hive.
```
spark-sql> SELECT CAST('1999 08 01' AS DATE);
NULL
```

@MaxGekk . Could you mention these two cases together in the PR title?

Reply:
I updated the PR description and JIRA.